pkg/volume/azure_dd/azure_provision.go

```diff
@@ -107,6 +107,7 @@ var _ volume.Provisioner = &azureDiskProvisioner{}
 
 func (a *azureDiskProvisioner) Provision() (*v1.PersistentVolume, error) {
 	var sku, location, account string
+	var FSType *string
```

Comment:
Im I remember correct, Azure already does the default fstype handling later and I did not want to touch that part. Azure was well tested by me at that time and it worked well.